### PR TITLE
Update Node.js client to use new SDK

### PR DESCRIPTION
**Pull request by jwulf**
## Description

Update the docs to use the new Node.js SDK.

Related to https://github.com/camunda/camunda-docs/issues/3819

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
I've added @camunda/tech-writers since Christina is on FTO.

**Comment by mesellings** [comment]
@jwulf I'll review this for you

**Review by mesellings — DISMISSED** [review]
Lgtm - happy to approve 👍

**Comment by akeller** [comment]
Cleaning this up for you @jwulf so we can merge.

File: versioned_docs/version-8.5/components/best-practices/development/writing-good-workers.md [hunk 1]
```diff
@@ -264,7 +264,7 @@ zbc.createWorker({
 
 This is **reactive code**. And a really interesting observation is that reactive programming is so deep in the JavaScript language that it is impossible to write blocking code, even code that looks blocking is still [executed in a non-blocking fashion](https://github.com/berndruecker/camunda-cloud-clients-parallel-job-execution/blob/main/results/nodejs-blocking.log).
 
-Node.js code scales pretty well and there is no specific thread pool defined or necessary. The Camunda 8 Node.js client library also [uses reactive programming internally](https://github.com/camunda-community-hub/zeebe-client-node-js/blob/master/src/zb/ZBWorker.ts#L28).
+Node.js code scales pretty well and there is no specific thread pool defined or necessary. The Camunda 8 Node.js client library also [uses reactive programming internally]([https://github.com/camunda-community-hub/zeebe-client-node-js/blob/master/src/zb/ZBWorker.ts#L28](https://github.com/camunda/camunda-8-js-sdk/blob/main/src/zeebe/zb/ZBWorker.ts#L27)).
```
[the remark this answers is not in the document]
Comment (akeller) [reply versioned_docs/version-8.5/components/best-practices/development/writing-good-workers.md:267]: I think you want this to point to L27

**Review by akeller — APPROVED** [review]

**Pull request merged**